docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -118,19 +118,45 @@ Follow the instructions below based on the query type:
 10. Click **Save**. To create a monitor, click [Save and Create Monitor](#creating-slo-monitors).
 
 
-## Creating SLOs from Logs, Metrics, Monitors
+## Create a Log-based SLO
 
 You can create SLOs directly from your Sumo Logic log search. This allows you to validate queries, quickly create SLOs, and re-use queries from existing dashboard panels.
 
 1. Enter a new **Log search** query (or use an existing one).<br/><img src={useBaseUrl('img/observability/log-search.png')} alt="log search" />
 1. Click the **More Actions** (kebab icon) dropdown menu.<br/><img src={useBaseUrl('img/observability/slo-more-actions-kebab.png')} alt="More Actions" />
 1. Click **Create an SLO**.<br/><img src={useBaseUrl('img/observability/slo-create.png')} alt="Create an SLO" width="200"/>
 
+
+## Create a Monitor-based SLO
+
+Critical Monitors are good candidates to convert to SLOs. As an example, if you have an existing Monitor that's set to alert you to Critical conditions. For example, in Kubernetes env, more than 5 pods in terminated state in 5-minute interval.
+
+From the Monitor UI, you can create a window-based SLO for a given trigger condition (like Critical) given time period 1m window duration. To create a Monitor-based SLO:
+
+:::important
+Your Monitor must be in an active state prior to creating a Monitor-based SLO.
+:::
+
+1. Open the alert panel > **More Actions** > **Create SLO**.
+1. In the **New SLO** window, you'll see that your Monitor conditions have auto-populated. You're welcome to revise and define additional SLI and SLO parameters here.
```

Comment:
Before this step user will need to select monitor based tab. Also we should add steps about selecting monitor and trigger types.
<img width="1536" alt="Screenshot 2023-01-25 at 3 32 23 PM" src="https://user-images.githubusercontent.com/87963972/214536946-7edeaed7-edad-4e9a-84d8-b7daf46b0bc4.png">

Feel free to play around on long data https://us1data.long.sumologic.net/ui/#/monitoring/slo/. All changes have reached this deployment.

Reply:
In your example above, it looks like none of the Monitor settings carried over... but in [this video](https://docs.google.com/presentation/d/16mduhRplWD43bog-Ohneg_DFxVWQc0FZk4Dz9Evpof8/edit#slide=id.g19cc821518d_2_45), all Monitor conditions are pre-populated?

Reply:
They do get carried over but only if you create the SLO via a Monitor i.e., go to a monitor -> click on it -> go to Monitor-based SLO section -> click Add Monitor-based SLO. In such a case Monitor and Trigger event will come as pre-filled.

The screenshot above is from a scenario where user is creating a new SLO from `SLOs` tab. Hence, system has no idea which monitor it should begin (or say auto populate) with.